src/utils/web3Utils.js

```diff
@@ -1,5 +1,11 @@
+import Web3 from 'web3';
 import getWeb3 from './getWeb3';
 
+const getAccounts = async (web3) => {
+    web3 = web3 || await getWeb3();
+    return await web3.eth.getAccounts();
+}
```

Comment:
I use a similar util function [here ](https://github.com/enabledao/enable-ui/blob/contract-integration-2/src/utils/web3Utils.js#L23)

Reply:
yea,
saw that.
Yours returns the expected default address, as against all addresses